coffee-chats/src/test/java/com/google/step/coffee/entity/ChatRequestBuilderTest.java

```diff
@@ -70,4 +75,13 @@ public void mustHaveValidChatDuration() {
     }
   }
 
+  @Test
```

Comment:
All that code required only a single test?

Reply:
The only backend code I wrote was fetching pre-existing entities, so I am not sure what to test other than the small behavioural change I made to ChatRequestBuilder, which I did so here. The large amount of code is largely front end component code to display the fetched data in a visually pleasing format.